apps/ledger-live-desktop/src/renderer/components/OperationsList/EditOperationPanel.tsx

```diff
@@ -21,14 +23,15 @@ const EditOperationPanel = (props: Props) => {
   const handleOpenEditModal = useCallback(() => {
     invariant(operation.transactionRaw, "operation.transactionRaw is required");
     dispatch(closeModal("MODAL_SEND"));
-    dispatch(
-      openModal("MODAL_EDIT_TRANSACTION", {
-        account,
-        parentAccount,
-        transactionRaw: operation.transactionRaw,
-        transactionHash: operation.hash,
-      }),
-    );
+    // dispatch(
```

Comment:
There is indeed a draft attempt to port this feature to evm here -> https://github.com/LedgerHQ/ledger-live/pull/4217

But since the changes commented out as part of this commit b9a7b5615cbc4f50218ac658b503777fa25ebe39 are not necessarily `ethereum` (or `evm`) specific (because present in non `ethereum` / `evm` family folders) I decided to only comment them as part of this PR.
Because these changes present in non family specific folders might not be present in the draft PR above (that only port the `ethereum` family logic to `evm`).

We could decide to remove all of the speed up / cancel tx code for now, until it's added back again in the evm family, but it should be done in it's own dedicated PR to allow easier revert and since it's outside the scope of this one (and also to avoid growing this already massive PR with even more code removal).
But removing all this code might not be strategically wise since the work to integrate speed up / cancel tx to evm should be (will be) done juste after this evm / ethereum family migration.

Reply:
Description updated 👍